src/config/detector-hints.config

```diff
@@ -694,6 +694,17 @@ NO DARK THEME
 
 ================================
 
+wikipedia.org
+
+TARGET
+html
+
+MATCH
+.skin-theme-clientpref-night
+.skin-theme-clientpref-os
```

Comment:
I can not accept with this class as you can end with white system theme and dark reader installed and used... then this will leave wikipedia on light mode.

I can accept the selector for permament night only.

Reply:
Please take a look at https://github.com/darkreader/darkreader/issues/11778 and https://github.com/darkreader/darkreader/pull/11995

Reply:
You want me to revert the #11995 for the auto selector? 🙄
In 2023 I wasn't experienced how the Dark Reader can work with Light Theme and that auto can end with light one and we should not disable Dark Reader for white one.

@alexanderby Can you tell how you think of adding detectors here for system-preffered?